include/RAJA/policy/cuda/reduce.hpp

```diff
@@ -274,8 +274,11 @@ RAJA_DEVICE RAJA_INLINE T block_reduce(T val, T identity)
 }
 
 
-template <typename GlobalIterationGetter, typename OP, typename T>
-RAJA_DEVICE RAJA_INLINE void grid_reduce(RAJA::expt::detail::Reducer<OP, T>& red)
+template <typename GlobalIterationGetter, template <typename, typename, typename> class OP, typename T>
```

Comment:
Is there a reason to pass Op into this as a template template parameter?

Reply:
This is the old implementation and I had forgotten to remove it. Should be gone now.